app/controllers/application_controller.rb

```diff
@@ -124,4 +124,13 @@ def store_user_location!
     # :user is the scope we are authenticating
     store_location_for(:user, originating_location)
   end
+
+  
+  def clear_flash_messages
```

Comment:
Would it be possible to have a quick doc string on this?

Reply:
When you say "hitting any action", what do you mean?  What are some of the actions you're preparing for?

Reply:
@daryanypl controller action method. def test_method end; If any flash messages persisted  just making null.

Reply:
<img width="1434" alt="Screen Shot 2020-10-09 at 1 17 26 PM" src="https://user-images.githubusercontent.com/30579702/95612804-f846d900-0a31-11eb-9749-8a56f2843df1.png">

Reply:
Please check console.log , Email is not null. Email is string "".

Reply:
Clear flash messages before hitting any controller method.